site/docs/connectors/runtime/open-lineage-janus-connector.md

```diff
@@ -0,0 +1,90 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Open Lineage Janus Connector
+
+The Open Lineage Janus connector allows the Open Lineages Services to connect with a JanusGraph database.
+
+The Connector can be configured to connect to either an embedded JanusGraph database or a standalone JanusGraph server.
+For more details on possible configurations please use the documentation [offered by JanusGraph](https://docs.janusgraph.org/basics/deployment/).
+
+Configuring the properties of the JanusGraph client when connecting to the database should be done following the [official documentation](https://docs.janusgraph.org/basics/configuration/).
+   
+--
+- Embedded JanusGraph
```

Comment:
Use heading 2 (md prefix ##) instead of bullet points it helps for better navigation.